src/common/KokkosKernels_SparseUtils_cusparse.hpp

```diff
@@ -0,0 +1,83 @@
+/*
+//@HEADER
+// ************************************************************************
+//
+//               KokkosKernels 0.9: Linear Algebra and Graph Kernels
```

Comment:
@lucbv the copyright update PR just went in, so you'll need to update the copyright header in this file, just pattern match another file